enterprise/coderd/coderd.go

```diff
@@ -352,6 +352,7 @@ func New(ctx context.Context, options *Options) (_ *API, err error) {
 				r.Use(
 					httpmw.ExtractProvisionerKeyParam(options.Database),
 				)
+				r.Get("/tags", api.fetchProvisionerKeyTags)
```

Comment:
I wonder if we're over-fitting to the ticket here. We could probably expand this endpoint to return all the details of the provisioner key, not just the tags. This would be more broadly applicable.

Reply:
Agreed - it would be more broadly useful in future.